### DISCUSSION TITLE
features/image-storage

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#24] [Announcements]
# features/image-storage

设置图床 内置图床 VanBlog 带有内置的图床实现，无需任何配置，开箱即用。 为防止更新后丢失数据，请在部署时确认目录映射正常。 第三方图床 VanBlog 可以对接第三方图床，是基于 picgo-core (https://picgo.github.io/PicGo-Core-Doc/) 实现的。 支持列表 七牛图床; 腾讯云 COS; 又拍云;...

https://vanblog.mereith.com/features/image-storage.html

**Comment by sunshang-hl** [comment]
建议增加使用cloudflare的教程,省去CDN的费用.

**Comment by finch04** [comment]
可以配置gitee图床吗
